pallets/author-mapping/src/lib.rs

```diff
@@ -248,42 +225,76 @@ pub mod pallet {
 		#[pallet::weight(<T as Config>::WeightInfo::set_keys())]
 		pub fn set_keys(origin: OriginFor<T>, keys: (NimbusId, T::Keys)) -> DispatchResult {
 			let account_id = ensure_signed(origin)?;
-			let old_author_id =
-				Self::nimbus_id_of(&account_id).ok_or(Error::<T>::OldAuthorIdNotFound)?;
-			let stored_info = MappingWithDeposit::<T>::try_get(&old_author_id)
+			let (new_author_id, keys) = keys;
```

Comment:
Can you replace author_id by nimbus_id ? It's more explicit

Reply:
should I do this everywhere in the pallet or just here?